vip_hci/psfsub/svd.py

```diff
@@ -581,6 +585,13 @@ def svd_wrapper(matrix, mode, ncomp, verbose, full_output=False,
             return S, V
         else:
             return U, S, V
+    elif left_eigv:
```

Comment:
There is no need to have different cases for `if left_eigv / else` when `full_outputs=True`  because `full_outputs` means that **both left and right eigvs are returned**. 

Nevertheless, I noticed an exception for the mode ('eigen', 'eigencupy', 'eigenpytorch') that returns only S and V when full_output is true.

Reply:
Among the 23 calls to `svd_wrapper`, 2 of them required `full_outputs=True` i;e `return U, S, V`. 
![image](https://user-images.githubusercontent.com/47563242/228528905-44925f6f-5685-46c0-830f-0589bfe9c7cd.png)


1- In svd.py (201): Compatible with returning U,S,V regardless of the chosen mode. 
![Capture d’écran 2023-03-29 135711](https://user-images.githubusercontent.com/47563242/228529185-2fc4b840-305a-4942-a567-009d215eb138.png)

2- In svd.py (668) : Only keep S anyways.
![Capture d’écran 2023-03-29 135632](https://user-images.githubusercontent.com/47563242/228529544-1dff9f70-61f0-471f-9403-037f566fc1cb.png)

What I suggest is to enable mode ('eigen', 'eigencupy', 'eigenpytorch') both left and right eigvs as well.

Reply:
If this was coded like this to avoid computing U, then I will add a `if` to not compute U if it is not necessary.

Reply:
Thanks for the checks! My worry was indeed regarding the exception for the 'eigen' modes.